src/Commands/Site/ImportCommand.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace Pantheon\Terminus\Commands\Site;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Terminus\Collections\Sites;
+use Terminus\Models\Environment;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+class ImportCommand extends TerminusCommand
+{
+    /**
+     * @var boolean True if the command requires the user to be logged in
+     */
+    protected $authorized = true;
```

Comment:
This style of annotation has been superseded by a superior method. You only need add `@authorized` hook to your command's doc block to activate this functionality.